docs/public_api.md

```diff
@@ -0,0 +1,139 @@
+Draft:
+Our subnet allows validators to run serve http (websocket) endpoint that allows external (external to subnet) clients to access the power (probably power is not the best word) of the subnet.
+
+# Short version
+To serve public endpoint you need to:
+1. Add `--public_api.enabled` to `neurons/validator.config.js`.
+2. Add `--public_api.server_port 8888` (default value) to `neurons/validator.config.js` and open the port.
+3. Add client API key with `scripts/api_key_cli.sh ~/.bittensor/miners/validator/default/netuid17/validator/api_keys.db add EWJzrqUL94r5BEz0ZLPUbEwD8qdQX18dCUF9B7C2e0 client-name 100 60`
+4. Wait for 30 minutes.
+
+# Configuring the validator 
+To start serving public endpoint, you need to turn it on. You need to run validator with `--public_api.enabled` flag.
+By default `--public_api.server_port` is set to 8888, you can change it and open corresponding port on your firewall.
+
+Other public API settings:
+`--public_api.copies` - validator will give the task to `--public_api.copies` miners and pick the best results. It's not recommended to change this settings. Default value 4 is considered a good balance between  latency and redundancy.
```

Comment:
The validator will assign the task...

Reply:
By default four miners are assigned to each task, which has shown to be a good balance between latency and redundancy